src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -363,6 +363,53 @@
                     }
                 }
             ]
+        },
+        {
+            "Id": "VPCConnector",
+            "Name": "VPC Connector",
+            "Description": "App Runner requires this resource when you want to associate your App Runner service to a custom Amazon Virtual Private Cloud (Amazon VPC).",
+            "Type": "Object",
+            "AdvancedSetting": true,
+            "Updatable": true,
+            "ChildOptionSettings": [
+                {
+                    "Id": "CreateNew",
+                    "Name": "Create New VPC Connector",
+                    "Description": "Do you want to create a new VPC Connector?",
+                    "Type": "Bool",
+                    "DefaultValue": false,
+                    "AdvancedSetting": false,
+                    "Updatable": true
+                },
+                {
+                    "Id": "Subnets",
+                    "Name": "Subnets",
+                    "Description": "A list of IDs of subnets that App Runner should use when it associates your service with a custom Amazon VPC. Specify IDs of subnets of a single Amazon VPC. App Runner determines the Amazon VPC from the subnets you specify.",
+                    "Type": "List",
```

Comment:
Should there be a sub property like `ListType` that indicates the type of data for the list. In this case `string`?

Reply:
Since there is no need for this at the moment, I would say let's stick with the assumption that by default a List type is a List<string>. In the future, if we add ListType to the recipe we could change the default from string to whichever type is specified. We wouldn't have any one way doors in this case.